### ISSUE TITLE
roxygen2 7.3.0 breaks NAMESPACE

### ISSUE THREAD

**Issue by jdblischak** [Merck/gsDesign2#313]
While working on #312, I noticed that the latest roxygen2 was deleting `:=` from {data.table} and `%>%` from {dplyr} from `NAMESPACE`.

```R
==> devtools::document(roclets = c('rd', 'collate', 'namespace'))

ℹ Updating gsDesign2 documentation
✖ pw_info.R:75: @importFrom Excluding unknown export in from data.table:
  `":="`.
✖ pwe.R:153: @importFrom Excluding unknown export in from dplyr: `"%>%"`.
ℹ Loading gsDesign2
✖ pw_info.R:75: @importFrom Excluding unknown export in from data.table:
  `":="`.
✖ pwe.R:153: @importFrom Excluding unknown export in from dplyr: `"%>%"`.
Documentation completed
```

roxygen2 7.3.0 was recently [released](https://www.tidyverse.org/blog/2024/01/roxygen2-7-3-0/) on 2024-01-11. Fortunately another developer has already reported the problem in https://github.com/r-lib/roxygen2/issues/1570. While we wait on a fix, please build the documentation with roxygen2 7.2.3

**Comment by nanxstats** [comment]
😂 Nice findings. Will surely keep this in mind... Thanks!

To (re-)install roxygen2 7.2.3, one can use:

```r
remotes::install_version("roxygen2", "7.2.3")
```

**Comment by olivroy** [comment]
Actually, you would fix those by replacing the lines with:

```r
@importFrom data.table :=
@importFrom dplyr %>%
```

**Comment by jdblischak** [comment]
> Actually, you would fix those by replacing the lines with:

@olivroy Thanks for the suggested workaround! While the roxygen2 developers are restoring the previous behavior in https://github.com/r-lib/roxygen2/pull/1574, I tested and confirmed that `@importFrom data.table :=` and `@importFrom data.table %>%` work as intended with both roxygen2 7.2.3 and 7.3.0.

I've always put quotes around these types of functions. This is how they display in `NAMESPACE`, and I thought I had got errors in the past when I didn't put them in quotes. Either I am misremembering, or roxgyen2 was updated to handle this common use case.
